checka11y.css

```diff
@@ -472,3 +472,15 @@ h6[aria-hidden]::after {
   content: "ERROR: headings should be reachable by assistive technologies (no aria-hidden)" !important;
   font-family: var(--checka11y-font, cursive);
 }
+
+/**
+* Accesskey attribute could interfere and conflict with screen readers and assistive technologies
+*/
+[accesskey] {
+  border: var(--checka11y-error-border-width, 5px) solid var(--checka11y-error-color, red) !important;
+}
+
+[accesskey]::after {
+  content: "ERROR: accesskey attribute could interfere and conflict with screen readers and assistive technologies" !important;
```

Comment:
@alvaromontoro Did you recommend this be a warning, rather than an error? 🙂

Reply:
It's a warning. Accesskey is not bad, just if the selected keys match the screen reader keys.